kratos/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ ENDFOREACH(subFolder ${ListFolders})
 ## Kratos python interface code
 FILE(GLOB_RECURSE KRATOS_PYTHON_SOURCES "python/*.cpp") #all .cpp
 LIST(APPEND KRATOS_PYTHON_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/processes/assign_scalar_field_to_entities_process.cpp)
+if(${KRATOS_BUILD_TESTING} MATCHES ON)
+    LIST(APPEND KRATOS_PYTHON_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp_tests/utilities/test_python_function_callback_utility.cxx) # We are using a different extension in order to avoid the GLOB_RECURSE for tests
+endif(${KRATOS_BUILD_TESTING} MATCHES ON)
```

Comment:
not sure abt this one

Can you make a python-test instead? To me looks like this test doesn't need C++

BTW whats the progress with the library?

Reply:
Do you mean to move it to the python folder?, I like more this approach so it is with the other tests, and the only thing I do is add it with a different extension

The library has been patched to be compatible with the CI (now works both with C++11 and previous versions, so in modern C++ there were many warnings due to missing override).

Reply:
What do you mean then @philbucher?, to move it to the `python` folder?

Reply:
no, make a python test instead, then we don't have this problem

Reply:
I meant to use Python as language for implementing the test, not C++ 
=> looking at the test the same can be done using python, in this case doesn't have to be C++
This was we will not have the trouble with the includes in C++ level

Reply:
I would like the test to be in C++ to ensure it works once the lib is changed

Reply:
But I can move it to python otherwise

Reply:
ok makes sense

hm not sure, probably better to not have it in the core, as then it would include python

@roigcarlo what do you think? This is a bit over my head

Reply:
@roigcarlo ?

Reply:
I tend to agree with @philbucher in this one.

If `python.h` or `pybind.h` are in the include tree of the test they cannot go into the core. If you want to have this test it should be included in a new `KratosInterfaceTests` target or something like that.

I would also suggest to add a comment or something indicating that a user should not use this inside Kratos under any circumstance. The use case for this test has a very narrow scope that we don't want to promote.

Reply:
Now?